src/Sulu/Bundle/PreviewBundle/Resources/config/routing.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<routes xmlns="http://symfony.com/schema/routing"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+
+    <route id="sulu_preview.start" path="/start">
+        <default key="_controller">sulu_preview.preview_controller:startAction</default>
+        <default key="_fos_rest_zone">true</default>
```

Comment:
What was our decision here? Maybe it is really worth thinking about setting the zone in the config to `/admin`?

Reply:
@chirimoya

Reply:
`/admin`